src/BlazorWebView/src/Maui/Windows/WinUIWebViewManager.cs

```diff
@@ -102,5 +115,30 @@ protected override void QueueBlazorStart()
 					");
 			};
 		}
+
+		private void Webview_CoreWebView2Initialized(WebView2Control sender, UI.Xaml.Controls.CoreWebView2InitializedEventArgs args)
+		{
+			_webview.CoreWebView2.NavigationStarting += CoreWebView2_NavigationStarting;
+			_webview.CoreWebView2.NewWindowRequested += CoreWebView2_NewWindowRequested;
+		}
+
+		private void CoreWebView2_NavigationStarting(CoreWebView2 sender, CoreWebView2NavigationStartingEventArgs args)
+		{
+			var uri = new Uri(args.Uri);
+			if (uri.Host != "0.0.0.0" && _externalLinkMode == ExternalLinkMode.OpenInExternalBrowser)
+			{
+				_ = Launcher.LaunchUriAsync(uri);
```

Comment:
Should we log errors thrown by these tasks or do they appear somewhere?

Reply:
Just searched through, there doesn't seem to be an existing logging mechanism. Is there a standardized mechanism we could use here (or perhaps https://github.com/dotnet/maui/issues/4441 would cover it?).